frontend/hooks/useStakingContracts.ts

```diff
@@ -28,14 +28,14 @@ export const useStakingContracts = () => {
         (acc: StakingProgramId[], stakingProgramId: StakingProgramId) => {
           if (!isActiveStakingProgramLoaded) return acc;
 
-          // Put the active staking program at the top
-          if (stakingProgramId === currentStakingProgramId)
-            return [stakingProgramId, ...acc];
-
           // If the program is deprecated, ignore it
           if (STAKING_PROGRAMS[evmHomeChainId][stakingProgramId].deprecated)
```

Comment:
it seems we forgot to add the logic where we show an alert on main that user should switch from deprecated contract to a new one... can you please add it quickly and confirm with Iason it looks fine?

here's it in legacy beta https://github.com/valory-xyz/olas-operate-app/blob/legacy/pearl-beta/frontend/components/MainPage/sections/AlertSections/ContractDeprecatedAlert.tsx 

usage here: https://github.com/valory-xyz/olas-operate-app/blob/legacy/pearl-beta/frontend/components/MainPage/sections/AlertSections/index.tsx#L22